src/app/list-manager/sidebar/sidebar.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="sidebar-container h-100 w-100 overflow-y-auto" [attr.data-mobile]="isMobile">
+    <div>
+        <input type="text" class="form-control w-100 rounded-0" [(ngModel)]="filterExpr" (keyup)="onChangeFilter()"
+            [placeholder]="i18n('listManager.filterPlaceholder')" autocomplete="off">
+    </div>
+    @for (noteItem of filteredLists; track $index) {
```

Comment:
Again: Angular 18 uses `@for` instead of `*ngFor` and `@if` for `*ngIf` and other control structures. Check the documentation at https://angular.dev/guide/templates/control-flow.
Do not suggest this change again.